pkg/accountmgmtutil/quota.go

```diff
@@ -0,0 +1,6 @@
+package accountmgmtutil
+
+const (
+	QuotaTrialType    = "eval"
```

Comment:
It is common practice to create a custom type for enums like this in Go. Example: `type QuotaType = string`